### ISSUE TITLE
Suggestion: Log Villager Interaction

### ISSUE THREAD

**Issue by domucmba** [PlayPro/CoreProtect#222]
Suggest add logging PlayerInteraction events with a villager either as core function or addon similar to CoreProtect TNT plugin

**Comment by Intelli** [comment]
To clarify, you just want to see that a user clicked on a villager, like if they click on a door?

(Villager trades are already logged under a:inventory)

**Comment by domucmba** [comment]
Yes. When a player trades with a villager the a:inventory events show that an item was removed from the player's inventory and then a new item was added but there isn't an indication that the item came from/when to a villager. If the click event which brings up the trading GUI were logged then you could inspect the villager to see who has traded with that villager.
